test/test_va_api_fixture.h

```diff
@@ -112,6 +112,25 @@ class VAAPIFixture : public ::testing::Test
     std::string m_skip;
 };
 
+/* Test fixture that initializes a shared display once per test suite, and
+ * provides it to you in m_vaDisplay from its SetUp() routine.
+ */
+class VAAPIFixtureSharedDisplay : public VAAPIFixture
+{
+public:
+    VAAPIFixtureSharedDisplay();
+
+    virtual void SetUp();
+
+    static void SetUpTestSuite();
+    static void TearDownTestSuite();
+
```

Comment:
Hmmm... what happens if a derived test of this subclass decides to call `getDisplay`, `doInitialize`, or `doTerminate` in a future change?  Perhaps those need to be overridden here, too.

Reply:
It would be pretty invalid to call those in the shared display fixture.  So you want to just have stubs that `FAIL()` to make sure someone doesn't accidentally do so?

Reply:
Note also that previous extra use of those functions before the shared test fixture would leak memory or double-free.

Reply:
I guess it's fine to leave this as is since current tests use them properly and the class isn't very robust to begin with.  We can harden the design later.